src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -425,6 +456,10 @@ class __attribute__((visibility("default"))) TvgLottieAnimation
 
 EMSCRIPTEN_BINDINGS(thorvg_bindings)
 {
+    #ifdef THORVG_WG_RASTER_SUPPORT
+        emscripten::function("initWgInstance", &initWgInstance);
```

Comment:
This design looks weird to me. May I ask why this is needed? The `wgpu` initialization seems to happen in one place at a specific time. If you need to reconsider the calling position, the `wg` engine initialization might need to be rearranged accordingly together.